application/templates/cms/edit_measure_page.html

```diff
@@ -179,16 +179,16 @@
 
         <div class="grid-row">
             <div class="column-full">
-                <h1 class="heading-large">{% if new %}Create{% else %}Edit{% endif %} measure</h1>
+                <h1 class="heading-large">{% if new %}Create{% else %}Edit{% endif %} page</h1>
                     {% if not new %}{{ form.db_version_id }}{% endif %}
                     {% if measure.publication_date %}
                         <h3 class="heading-medium">Publication date: {{ measure.publication_date | format_friendly_date}}</h3>
                     {% endif %}
 
-                    {{ render_field(form.title, disabled=form_disabled, diffs=diffs) }}
+                    {{ render_field(form.title, disabled=form_disabled, diffs=diffs, hint="For example, ‘Victims of crime’") }}
 
                     <div class="form-group">
-                        <label class="form-label" for="subtopic">Sub-topic</label>
+                        <label class="form-label" for="subtopic">Choose a topic</label>
```

Comment:
Adding the word "choose" here implies a bit more agency than the user will generally have - if people are working on an update rather than a brand new measure (which is the case more often than not these days) then the topic is already selected and the select is disabled.  So in most cases the topic is already chosen and can't be changed on this form.

Reply:
Okay, please can we change it to 'Topic' - thx :)

Reply:
👍